test/libsolidity/GasTest.cpp

```diff
@@ -83,42 +67,28 @@ GasTest::GasTest(string const& _filename)
 
 void GasTest::parseExpectations(std::istream& _stream)
 {
-	std::map<std::string, std::string>* currentKind = nullptr;
-	std::string line;
+	map<std::string, std::string>* currentKind = nullptr;
+	string line;
 
 	while (getline(_stream, line))
-		if (boost::starts_with(line, "// creation:"))
+		if (!boost::starts_with(line, "// "))
+			BOOST_THROW_EXCEPTION(runtime_error("Invalid expectation: expected \"// \"."));
+		else if (boost::ends_with(line, ":"))
 		{
-			auto it = line.begin() + 12;
```

Comment:
What are these changes?

Reply:
If we only test in release mode, we don't need to parse the integer value of the creation cost, but we can treat the ``creation`` part exactly like ``internal`` and ``external`` (i.e. just use string comparison). When I wrote this earlier I expected that we would adjust the creation gas cost values depending on the release/prerelease mode, but if we end up not doing that, this can be simplified.